src/Maestro/MaestroApplication/ApplicationParameters/Local.1Node.xml

```diff
@@ -18,4 +18,4 @@
     <Parameter Name="PullRequestActorService_MinReplicaSetSize" Value="1" />
     <Parameter Name="FeedCleanerService_InstanceCount" Value="1" />
   </Parameters>
-</Application>
+</Application>
```

Comment:
nit: Something stripped the newline on the EOFs here, this makes github and some folks' tooling complain, consider reverting.

Reply:
Yeah, I saw that it made changes to these SF files, so I reverted the changes. Thanks Visual Studio :P